packer/scripts/ubuntu/update.sh

```diff
@@ -16,12 +16,16 @@ sed -i.bak 's/^Prompt=.*$/Prompt=never/' /etc/update-manager/release-upgrades;
 apt-get -y update;
 
 # update package index on boot
-cat <<EOF >/etc/init/refresh-apt.conf;
-description "update package index"
-start on networking
-task
-exec /usr/bin/apt-get update
-EOF
+# Upstart not incl in 20.04 - init dir no longer present in 20.04
+# Does this need to continue to exist? If so, this file needs to be created in
+# a different location.
+
+#cat <<EOF >/etc/init/refresh-apt.conf;
```

Comment:
See comment - does this need to be kept? If so, we need to find an alternate location to write this file where it will be executed.